src/maidsafe/vault/mpid_manager/service.h

```diff
@@ -127,14 +133,25 @@ class MpidManagerService {
   void HandleSyncedRemoveAccount(
            std::unique_ptr<MpidManager::UnresolvedRemoveAccount>&& synced_action);
 
+// =============== Account Transfer ==============================================================
+  void TransferAccount(const NodeId& destination,
+                       const std::vector<MpidManager::KVPair>& accounts);
+  void HandleAccountTransferEntry(const std::string& serialised_account,
+                                  const routing::SingleSource& sender);
+  void HandleAccountTransfer(const MpidManager::KVPair& account);
+  void HandleAccountQuery(const ImmutableData::Name& name,
+                          const NodeId& sender,
+                          const NodeId& receiver);
+
   routing::Routing& routing_;
-  mutable std::mutex accumulator_mutex_, nodes_change_mutex_, pending_account_mutex_;
+  mutable std::mutex accumulator_mutex_, nodes_change_mutex_, pending_account_mutex_, mutex_;
```

Comment:
I prefer leave it as it is in this pull request.
When the code base got stable and we had a decided policy,
starting a dedicated task to change it.